optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -12,40 +17,108 @@ const PreferentialTrial: FC<{
   const theme = useTheme()
   const action = actionCreator()
   const trialWidth = 500
+  const trialHeight = 300
+  const [hover, setHover] = useState(false)
```

Comment:
Could you please avoid to use state for the performance reason? We should write the animation in CSS.